index.html

```diff
@@ -203,11 +213,13 @@ <h4>About me</h4>
 <!---------  contact form ------------------------------------->
 <form action="https://formspree.io/f/mqkoavny" method="post">
```

Comment:
- [ ] Please adjust the button alignment in the form to match the template. 🥸️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/214768852-2aafc28f-5275-4042-913d-7302b33d2bf6.png" width="400">|<img src="https://user-images.githubusercontent.com/17243325/214768847-5eb01ac8-218e-41ec-b543-d3af42f7a3ca.png" width="400">|